pkg/tanka/parallel.go

```diff
@@ -54,19 +54,19 @@ func ParseParallel(paths []string, opts ParseParallelOpts) (envs []*v1alpha1.Env
 	}
 
 	if len(allErrors) != 0 {
-		return envs, ErrParseParallel{errors: allErrors}
+		return envs, ErrParallel{errors: allErrors}
 	}
 
 	return envs, nil
 }
 
-type parseJob struct {
+type parallelJob struct {
 	path string
 	opts JsonnetOpts
 	env  *v1alpha1.Environment
 }
 
-func parseWorker(envsChan <-chan parseJob) (errs []error) {
+func parallelWorker(envsChan <-chan parallelJob) (errs []error) {
```

Comment:
(missed this in the last review of this code)

The pointer modification in L76 is surprising to me, because it circumvents the channel communication style of Goroutines.
A more obvious way to handle that would be to have the worker accept a channel it sends results into (env & err), and read from that result channel as often as envs exist in `parallelLoadEnvironments`

See https://github.com/grafana/tanka/blob/9facf1f8e137f069aa0f274a423fee7fe30196af/pkg/tanka/list.go#L63-L88 for an example of that

Reply:
Only review cosmetic changes in this PR please.

Reply:
This one actually is cosmetic :P